Cargo.toml

```diff
@@ -68,6 +68,13 @@ serde_json = "1.0"
 tempfile = "3.2"
 tokio = {version = "1.0", features = ["macros", "rt-multi-thread", "time", "process"]}
 tonic = "0.4"
+tokio-stream = { version="0.1", features = ["fs", "net"] }
```

Comment:
Do these deps not work in `dev-dependencies`?

Reply:
They are under `dev-dependencies` i believe

Reply:
Ugh...I thought this was in the other cargo file for krustlet. Ignore me